Application/People/Meta/Common/Service/Entity/ViewPeopleMetaCommon.php

```diff
@@ -200,4 +200,7 @@ public function getTblCommonInformation_IsAssistance()
         }
         return $this->TblCommonInformation_IsAssistance ? 'Ja' : 'Nein';
     }
+
+    /** @return string */
+    public function getBirthplace() { return $this->TblCommonBirthDates_Birthplace; }
```

Comment:
Codacy found an issue: [Opening brace should be on a new line](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)